crates/astria-sequencer-relayer/src/data_availability.rs

```diff
@@ -160,6 +160,7 @@ impl NamespaceData for SequencerNamespaceData {}
 #[derive(Serialize, Deserialize, Debug)]
 pub struct RollupNamespaceData {
     pub(crate) block_hash: Hash,
+    #[serde(with = "hex::serde")]
```

Comment:
Note that this was serialize as a json list of integers. Makes sense to change the deser format if everything else is being changed as well.